contrib/legal/privacy.html.sample

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Privacy Policy</title>
+ </head>
+
+ <body>
+  <h1>Privacy Policy</h1>
+
+  <i>Last updated: December 28, 2019</i>
```

Comment:
`<i>` is an inline element not a block level element.

Use a `<h3>` or `<h4>` etc.

```suggestion
  <h4>Last updated: December 28, 2019</h4>
```